### ISSUE TITLE
 No module named 'requests'

### ISSUE THREAD

**Issue by root-ansh** [bombardier-gif/covid-vaccine-booking#70]
hi i was trying to install this script in mac os 11.2.1 big sur , i followed all the steps but while running the script i get this error "no module named requests" . do we have to install some additional packages? is it via brew or pip?

**Comment by yashk2103** [comment]
yes buddy additional packages are required. Check requirements.txt and install it using pip. Check Readme.md
